Packs/Jira/Scripts/JiraCreateIssueExample/JiraCreateIssueExample_test.py

```diff
@@ -1,82 +1,134 @@
     Then:
-        - dict with known argument key is returned
-        - dict with known argument key is returned
-        - empty dict
+        - Case A: No exception is thrown.
+        - Case B: A `ValueError` exception is thrown.
     """
 
-    args = get_known_args_from_input(input)
+    try:
+        validate_date_field(due_date)
+    except ValueError as ve:
+        assert "unconverted data remains" in str(ve)
+
 
-    assert isinstance(args, dict)
-    assert args == expected
+@pytest.mark.parametrize("due_date", [
+    ("2022-1-1"),
+    ("2022-31-31")
+])
+def test_validate_date_field_format(due_date):
 
+    """
+    Given:
+        - A date.
+
+    When:
+        - Case A: The date is in valid format.
+        - Case B: The date has an invalid month.
+
+    Then:
+        - Case A: No exception is thrown.
+        - Case B: A `ValueError` exception is thrown.
+    """
 
-@pytest.mark.parametrize("input,expected", [
-    ({DEFAULT_ARGS[0]: "system down"}.items(), {}),
-    ({DEFAULT_ARGS[0]: "system down", "unknown_key": "some key"}.items(), {"unknown_key": "some key"}),
-    ({"unknown_key": "some key"}.items(), {"unknown_key": "some key"})
+    try:
+        validate_date_field(due_date)
+    except ValueError as ve:
+        assert f"time data '{due_date}' does not match format '{DATE_FORMAT}'" in str(ve)
+
+
+@pytest.mark.parametrize("custom_fields, expected", [
+    (["customfield_10096=test"], [{"customfield_10096": "test"}]),
+    (["customfield_10096=test", "customfield_10040=100"], [{"customfield_10096": "test"}, {"customfield_10040": 100}]),
+    (["customfield_10096=test", "customfield_10040=0100"], [{"customfield_10096": "test"}, {"customfield_10040": 100}]),
+    (["customfield_10096=test", "customfield_10040=A100"], [{"customfield_10096": "test"}, {"customfield_10040": "A100"}]),
+    (["customfield_10096:test", "customfield_10040=A100"], [{"customfield_10040": "A100"}]),
+    ([], []),
 ])
-def test_get_additional_args_from_input(input, expected):
+def test_parse_custom_fields(custom_fields, expected):
+
     """
     Given:
-        - A dictionary view with Jira fields
+        - A list of strings of custom fields.
+        - An expected list of dicts of custom fields.
+
     When:
-        - 1 unknown argument key is supplied
-        - 1 unknown argument key is supplied, 1 known argument key is supplied
-        - 1 known arguments are supplied, 0 unknown argument key is supplied
+        - Case A: 1 text custom field.
+        - Case B: 1 text custom field, 1 integer custom field.
+        - Case C: 1 text custom field, 1 integer custom field with 0 padding.
+        - Case D: 2 text custom fields.
+        - Case E: 1 text custom field, 1 custom field with unexpected delimiter (:).
+        - Case F: Empty custom field list.
 
     Then:
-        - empty dict
-        - dict with unknown argument key is returned
-        - dict with unknown argument key is returned
+        - Case A: 1 text custom field returned.
```

Comment:
Then should be `Verify/assert that the given key and values separated by an '=' sign are returned as key and value inside a dict` or any phrasing you see fit.
So are the rest of the cases.

Reply:
Addressed in [`2b875a6` (#22601)](https://github.com/demisto/content/pull/22601/commits/2b875a6111d34fc492e18ac7ee5335e81c46a278).